pkg/katautils/config.go

```diff
@@ -1218,7 +1218,7 @@ func LoadConfiguration(configPath string, ignoreLogging, builtIn bool) (resolved
 	}
 
 	config.Trace = tomlConf.Runtime.Tracing
-	tracing = config.Trace
+	Tracing = config.Trace
```

Comment:
If we set the `tracingSet` variable in pkg/katautils/katatrace/tracing.go here, we can remove `tracing bool` from the `CreateTracer()` parameters.

Reply:
Fixed